src/brave-talk.ts

```diff
@@ -10,3 +10,11 @@ function generateRoomWithoutSeparator() {
     .replace(/\//g, "_")
     .replace(/=/g, "");
 }
+
+export function createRoom(roomUrl: string) {
+  window.open(
```

Comment:
should probably set `noopener` and `noreferrer` here https://developer.mozilla.org/en-US/docs/Web/API/Window/open

Reply:
good suggestion, updated